optoforce/README.md

```diff
@@ -14,6 +14,14 @@ The driver relies on the pySerial library to get data from the sensors through U
 
 The ROS node built with this driver publishes messages of type `geometry_msgs/WrenchStamped` for every sensor.
 
+## Add your user to the dialout group
+Add yourself to the group *dialout* in order to get access the serial port.
+You may use
+
+`sudo adduser $USER dialout`.
```

Comment:
I would rather use

`sudo usermod -aG $USER dialout`

I think adduser will try to create a new user.